explorer/src/lib/components/blocks-card/BlocksCard.svelte

```diff
@@ -14,6 +14,9 @@
   /** @type {Boolean} */
   export let loading;
 
+  /** @type {AppStore} */
+  export let appStore;
```

Comment:
I think it just need to receive the `isSmallScreen` property here (like in the latest blocks / transactions): if the store is needed it should just import it.